src/wandbot/chat/rag.py

```diff
@@ -58,12 +62,51 @@ def __init__(
         )
         self.response_synthesizer = ResponseSynthesizer()
 
+    def generate_multi_modal_initial_response(
+        self, question: str, images: List[str]
+    ) -> str:
+        model = ChatOpenAI(model="gpt-4-vision-preview", max_tokens=500)
+        system_message = SystemMessage(
+            content="""You are a Weights & Biases support expert.
+            Your goal is to answer the user's question and provide them with the best possible solution.
+            You are provided with a support ticket and a set of screenshots related to the issue.
+            Provide a detailed solution to the user query based on the ticket and the screenshots."""
```

Comment:
@soumik12345 Change the prompt to

```
You are a Weights & Biases support expert. Your goal is to describe the attached screenshots in the context of the user query. You are provided with a support ticket and screenshots related to the issue. Provide a detailed description of the image in the context of the query so that the ticket can be answered correctly while incorporating the image info.
```